tests/integration/test_bundle.py

```diff
@@ -42,23 +45,23 @@ def get_bundle_template() -> Path:
     return get_this_script_dir() / ".." / ".." / "bundle.yaml.j2"
 
 
-async def get_unit_address(ops_test: OpsTest, app_name: str, unit_num: int) -> str:
-    """Get private address of a unit."""
-    status = await ops_test.model.get_status()  # noqa: F821
-    return status["applications"][app_name]["units"][f"{app_name}/{unit_num}"]["address"]
-
-
-async def get_app_address(ops_test: OpsTest, app_name: str) -> str:
-    """Get address of an app."""
-    status = await ops_test.model.get_status()  # noqa: F821
-    return status["applications"][app_name]["public-address"]
+async def get_k8s_service_address(ops_test: OpsTest, service_name: str, client: Client) -> Optional[str]:
+    """Get the address of a LoadBalancer Kubernetes service using kubectl."""
+    try:
+        result = client.get(Service, name=service_name, namespace=ops_test.model.name)
```

Comment:
(If I understand it correctly) I guess the client should be the `AsyncClient`.

Reply:
Another thing is: could we make the following as fixtures?

- `get_k8s_service_address`
- `get_reverse_proxy_app_url`

So that we don't have to call it everywhere explicitly, and avoid passing arguments through different call stacks (e.g. the `client` instance)